src/Dapr.Actors/Runtime/ActorTimerManager.cs

```diff
@@ -0,0 +1,43 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+using System.Threading.Tasks;
+
+namespace Dapr.Actors.Runtime
+{
+    /// <summary>
+    /// Provides timer and reminder functionality to an actor instance.
+    /// </summary>
+    public abstract class ActorTimerManager
```

Comment:
New abstraction for mocking/testing. This isolates the user-level APIs from the implementation (facade).